### DISCUSSION TITLE
Cannot Download MSI for WSL 2

### DISCUSSION THREAD

**Discussion by MaxRoeder** [microsoft/WSL#5946] [General] [answered]
I have currently WSL Version 1 on my Computer. When i try to update it, it references me to aka.ms/wsl2kernel where i should be able to Download the new Update/Kernel. The problem is i cant find the Download link there. How can i get the MSI or the Update now?

**Comment by mattwojo** [comment, marked as answer]
Looking into this right now -- looks like our localization team hasn't picked up the change yet. Apologies! 
Here's the links in the meantime:
[WSL2 Linux kernel update package for x64 machines](https://wslstorestorage.blob.core.windows.net/wslblob/wsl_update_x64.msi)

[WSL2 Linux kernel update package for ARM64 machines](https://wslstorestorage.blob.core.windows.net/wslblob/wsl_update_arm64.msi)

**Reply by craigloewen-msft** [reply]
And we added a temporary fix for it to our docs: https://github.com/MicrosoftDocs/WSL/pull/933

**Reply by craigloewen-msft** [reply]
This aka ms link is fixed now, and we are working on getting the changes translated over now.
